lib/active_storage/send_zip_helper.rb

```diff
@@ -13,15 +13,15 @@ module SendZipHelper
     #
     # @param files [ActiveStorage::Attached::One|ActiveStorage::Attached::Many|Array|Hash] file(s) to save
     # @return [String] folder path of saved files
-    def self.save_files_on_server(files)
+    def self.save_files_on_server(files, resize_to_limit: nil)
```

Comment:
Can you add documentation about this params

~~~rb
# @param [Boolean] Allow resizing of files before downloading
~~~

Reply:
@madeindjs hi! Yes, I will make the requested changes.